daemon/api_users.go

```diff
@@ -333,11 +333,18 @@ func createUser(c *Command, createData postUserCreateData) Response {
 			return InternalError("cannot create user: cannot get model assertion: %v", err)
 		}
 	}
+	var serial *asserts.Serial
```

Comment:
this should go under the if for Known no?

Reply:
We have two `if createKnown` below, one for the generial "create-all-known" case and one for the create-known with a specific email case. Should I still move the code into the two places? Or am I misunderstanding the suggestion?

Reply:
it should go under the same if as where we get model, sorry, it was clear to me